accounting/localizations/mexico.rst

```diff
@@ -262,12 +262,40 @@ If the invoice is not paid.
 Payments (Just available for CFDI 3.3)
 --------------------------------------
 
-To generate the payment complement you just must to follow the normal payment
+To generate the payment complement you only must to follow the normal payment
 process in Odoo, this considerations to understand the behavior are important.
 
-1. All payment done in the same day of the invoice will be considered as It
-   will not be signed, because It is the expected behavior legally required
-   for "Cash payment".
+1. To generate payment complement the payment term in the invoice must be
+   PPD, because It is the expected behavior legally required for
+   "Cash payment".
+
+   **1.1. How can I generate an invoice with payment term `PUE`?**
+
+   `According to the SAT documentation`_ if the payment was agreed that the
+   amount total that covers the document will be received before that the
+   day 17 of the month calendar immediately after of the month that was
+   issued the CFDI.
+
+   **1.2. How can I get this with Odoo?**
+
+   In Odoo is very easy to indicate when an invoice must be PPD or PUE, is
+   based on the `Payment Terms` defined in the invoice.
+
+   - If I do not define a `Payment Term` in the invoice the payment term must
+     be `PPD`.
+
+   - If today is the first day of the month and I generate an invoice with the
+     `Payment Term` `30 Net Days` I could see that the `Due Date` is less than
+     the day 17 of the next month, then the payment term is `PPD`.
```

Comment:
...then the payment term is `PUE`.

Maybe add that the payment term is for one payment (no partialities)

Reply:
Follow this example #310 (comment)